libs/azure-storage/README.md

```diff
@@ -1,9 +1,147 @@
 # langchain-azure-storage
 
-This package contains the LangChain integrations for Azure Storage. This project is a work-in-progress and currently does not contain any features.
+This package contains the LangChain integrations for Azure Storage. Currently, it includes:
+- `AzureBlobStorageLoader` - Loads LangChain document objects from Azure Blob Storage
 
 ## Installation
 
 ```bash
 pip install -U langchain-azure-storage
 ```
+
+## Document Loader Examples
+
+### Load from container
+Below shows how to load documents from all blobs in a given container in Azure Blob Storage:
+
+```python
+from langchain_azure_storage.document_loaders import AzureBlobStorageLoader
+
+loader = AzureBlobStorageLoader(
+    account_url=<your_account_url>,
+    container_name=<your_container_name>,
+)
+
+for doc in loader.lazy_load():
+    print(doc.page_content)
```

Comment:
We should maybe add a small in-line comment saying that this will print out the blob content as utf-8 text to re-enforce how these documents come back.